src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -144,82 +176,61 @@ private function dumpSitemap(PortalInformation $portalInformation)
      */
     private function dumpProviderSitemap($alias, PortalInformation $portalInformation)
     {
-        $provider = $this->sitemapProviderPool->getProvider($alias);
-        $maxPage = $provider->getMaxPage();
+        $page = 1;
+        do {
+            $sitemap = $this->sitemapRenderer->renderSitemap(
+                $alias,
+                $page,
+                $portalInformation->getLocale(),
+                $portalInformation->getPortal(),
+                $portalInformation->getHost(),
+                $this->scheme
```

Comment:
The scheme is passed as option, right? Shouldn't we configure that somewhere else? Otherwise forgetting an argument in the command will create a wrong sitemap without the user executing the command realizing it.